.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,22 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
```

Comment:
This file has been added and contains the configuration to do development inside a Docker container.